components/ws-manager-mk2/controllers/suite_test.go

```diff
@@ -50,10 +49,9 @@ func TestAPIs(t *testing.T) {
 }
 
 var (
-	ctx        context.Context
-	cancel     context.CancelFunc
-	wsActivity *activity.WorkspaceActivity
-	wsMetrics  *controllerMetrics
+	ctx       context.Context
+	cancel    context.CancelFunc
+	wsMetrics *controllerMetrics
```

Comment:
Metrics will be scrapped from all available replicas

Reply:
this will break some of the dashboards then (and maybe alerts too). For instance we sum all workspaces by cluster, this will then double the amount of reported workspaces

Reply:
thanks for the gists. Both replicas report workspace metrics, e.g.:
```
gitpod_ws_manager_mk2_workspace_phase_total{class="g1-standard",phase="Running",type="Regular"} 1
```
is in both. Summing them as done in dashboards/alerts would give 2 replicas, while there's only 1

Reply:
gist updated after running the integration test